overlays/bootstrap.nix

```
@@ -182,6 +182,7 @@ in {
                 ++ final.lib.optionals final.hostPlatform.isDarwin
                   (fromUntil "8.10.5" "8.10.6" ./patches/ghc/ghc-8.10.5-darwin-allocateExec.patch)
                 ++ until              "8.10.6" ./patches/ghc/Sphinx_Unicode_Error.patch
+                ++ fromUntil "8.10"   "8.10.8" ./patches/ghc/ghc-8.10-nonmoving.patch
```

Can we add the relevant links to the upstream Merge Requests?
```suggestion
                # this patch contains 
                # https://gitlab.haskell.org/ghc/ghc/-/merge_request/NNNN
                # https://gitlab.haskell.org/ghc/ghc/-/merge_request/NNNN
                # ...
                ++ fromUntil "8.10"   "8.10.8" ./patches/ghc/ghc-8.10-nonmoving.patch
```
You could also separate them out and name them
`./patches/ghc/MR<NNNN>-name.patch`

This will make it easer to understand what we do here.

---

Sure sounds good. I've separated out the MRs into their own patches